easybuild/easyconfigs/g/grid/grid-20220610-intel-2022a.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'PythonPackage'
+
+name = 'grid'
+version = '20220610'
+local_commit = '0677fc7'
+
+homepage = 'https://grid.qcdevs.org/'
+description = """Grid is a free and open-source Python library for numerical integration,
+                interpolation and differentiation of interest for the quantum chemistry community."""
+
+toolchain = {'name': 'intel', 'version': '2022a'}
+
+source_urls = ['https://github.com/theochem/grid/archive/']
+sources = ['%s.tar.gz' % local_commit]
+checksums = ['d8189c33a9d5fc89ad0fa83089431944b346686519ec8d3c350848cf28b81d7a']
+
+dependencies = {
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('sympy', '1.11.1'),
```

can we drop this to `1.10.1` if possible?
```suggestion
    ('sympy', '1.10.1'),
```